tests/DotNotationHelperTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\BLT\Tests\Helpers;
+
+use Blumilk\BLT\Helpers\DotNotationHelper;
+use PHPUnit\Framework\TestCase;
+
+class DotNotationHelperTest extends TestCase
+{
+    public function testGetValueUsingDotNotation(): void
+    {
+        $array = [
+            "user" => [
+                "profile" => [
+                    "email" => "user@example.com",
+                    "age" => 25,
+                ],
+            ],
+        ];
+
+        $this->assertEquals("user@example.com", DotNotationHelper::getValueUsingDotNotation($array, "user.profile.email"));
```

Comment:
Why not use Laravel's `[data_get()](https://laravel.com/docs/11.x/helpers#method-data-get)`? You could move `illuminate/support` from dev dependecies to dependencies in `composer.json` and use it directly here.